terraform/modules/happy-stack-eks/happy_config.tf

```diff
@@ -1,5 +1,5 @@
 data "happy_resolved_app_configs" "configs" {
-  app_name    = local.secret["tags"]["project"]
+  app_name    = len(var.app_name) == 0 ? local.secret["tags"]["project"]: var.app_name
```

Comment:
@hspitzley-czi how's this for a compromise?

Reply:
looks good to me